include/clang/Frontend/Utils.h

```diff
@@ -67,6 +69,18 @@ void InitializePreprocessor(Preprocessor &PP, const PreprocessorOptions &PPOpts,
                             const PCHContainerReader &PCHContainerRdr,
                             const FrontendOptions &FEOpts);
 
+/// DefineTypeSize - An overloaded helper that uses TargetInfo to determine
+/// the width, suffix, and signedness of the given type
+void DefineTypeSize(const Twine &MacroName, TargetInfo::IntType Ty,
+                           const TargetInfo &TI, MacroBuilder &Builder);
```

Comment:
Seems like the formatting is off. Did you run it through clang-format ?

Reply:
Fixed the formatting. There were other unrelated changes done by clang-format that I didn't commit.